spec/features/create_and_deliver_digests_spec.rb

```diff
@@ -0,0 +1,201 @@
+require "rails_helper"
+
+RSpec.describe "creating and delivering digests", type: :request do
+  before do
+    Timecop.freeze "2017-01-02 10:00"
+
+    allow_any_instance_of(DeliveryRequestService)
+      .to receive(:provider_name).and_return("notify")
+  end
+
+  after do
+    Timecop.return
+  end
+
+  scenario "daily digest run" do
+    login_with_internal_app
+
+    #create two subscriber lists with different links
+    list_one_topic_id = "0eb5d0f0-d384-4f27-9da8-3f9e9b22a820"
+    list_two_topic_id = "a915e039-070b-4633-813d-187af61cad7a"
+
+    stub_govdelivery("TEST123")
+    create_subscribable(title: "Subscriber list one", links: {
+      topics: [list_one_topic_id]
+    })
+
+    stub_govdelivery("TEST456")
+    create_subscribable(title: "Subscriber list two", links: {
+      topics: [list_two_topic_id]
+    })
+
+    lookup_subscribable("TEST123")
+    subscribable_one_id = JSON.parse(response.body)["subscribable"]["id"]
+
+    lookup_subscribable("TEST456")
+    subscribable_two_id = JSON.parse(response.body)["subscribable"]["id"]
+
+    #create two daily subscribers, one subscribed to daily digests for both
+    #subscribables and the other for daily for subscribable_one only
+    subscriber_one_address = "test-one@example.com"
+    subscriber_two_address = "test-two@example.com"
+
+    subscribe_to_subscribable(
+      subscribable_one_id,
+      address: subscriber_one_address,
+      frequency: Frequency::DAILY
+    )
+
+    subscribe_to_subscribable(
+      subscribable_two_id,
+      address: subscriber_one_address,
+      frequency: Frequency::DAILY
+    )
+
+    subscribe_to_subscribable(
+      subscribable_one_id,
+      address: subscriber_two_address,
+      frequency: Frequency::DAILY
+    )
+
+    #publish two items to each list
+    Timecop.freeze "2017-01-01 10:00" do
+      create_content_change(
+        title: "Title one",
+        content_id: SecureRandom.uuid,
+        description: "Description one",
+        change_note: "Change note one",
+        public_updated_at: "2017-01-01 10:00:00",
+        links: {
+          topics: [list_one_topic_id]
+        }
+      )
+    end
+
+    Timecop.freeze "2017-01-01 10:00" do
```

Comment:
The date here doesn't match the given `public_updated_at`, not sure if that matters though.

Reply:
Fixed in  208d127